applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -19,6 +21,10 @@
 from volume_source_test import VolumeSourceTest
 from fluid_analysis_test import FluidAnalysisTest
 from adjoint_fluid_test import AdjointFluidTest
+from adjoint_vms_element_2d import AdjointVMSElement2D
+from adjoint_vms_sensitivity_2d import AdjointVMSSensitivity2D
+from hdf5_io_test import HDF5IOTest
```

Comment:
I would use a better name here, this made me think its unrelated to fluid

Reply:
Well, we are actually testing the HDF5 application (we want to keep a test here to check the integration).

Reply:
How abt sth like HDF5IOWithFluid then?